scripts/TunnelAzure.ps1

```diff
@@ -129,24 +129,27 @@ Function Remove-SSHRule {
 
 Function New-TunnelVM {    
     Write-Header "Creating VM '$($Context.VmName)'..."
-    az vm create --location $Context.Location --resource-group $Context.ResourceGroup --name $Context.VmName --image $Context.Image --size $Context.Size --ssh-key-values "$($Context.SSHKeyPath).pub" --public-ip-address-dns-name $Context.VmName --admin-username $Context.Username --nics $Context.TunnelNicName --only-show-errors | Out-Null
+    az vm create --location $Context.Location --resource-group $Context.ResourceGroup --name $Context.VmName --image $Context.Image --size $Context.Size --ssh-key-values "$($Context.SSHKeyPath).pub" --public-ip-address-dns-name $Context.VmName --admin-username $Context.Username --nics $Context.TunnelNicName --enable-auto-update --patch-mode AutomaticByPlatform --only-show-errors | Out-Null
+    az vm update --name $Context.VmName --resource-group $Context.ResourceGroup --set osProfile.linuxConfiguration.patchSettings.assessmentMode=AutomaticByPlatform # Set the VM to periodically check for updates (once every 24 hours).
 
     if ($Context.BootDiagnostics) {
         Write-Header "Enabling boot diagnostics..."
         az vm boot-diagnostics enable --resource-group $Context.ResourceGroup --name $Context.VmName
     }
 }
 
-Function New-ServiceVM {    
-    Write-Header "Creating VM '$($Context.VmName)-server'..."
+Function New-ServiceVM { 
+    $vmName = "$($Context.VmName)-server"  
+    Write-Header "Creating VM '$vmName'..."
     if ($Context.WithIPv6) {
         # Create a VM with our NIC.
-        az vm create --location $Context.Location --resource-group $Context.ResourceGroup --name "$($Context.VmName)-server" --image $([Constants]::ServerVMImage) --size $Context.Size --ssh-key-values "$($Context.SSHKeyPath).pub" --admin-username $Context.Username --only-show-errors --nics $Context.ServiceNicName | Out-Null
+        az vm create --location $Context.Location --resource-group $Context.ResourceGroup --name $vmName --image $([Constants]::ServerVMImage) --size $Context.Size --ssh-key-values "$($Context.SSHKeyPath).pub" --only-show-errors --admin-username $Context.Username --enable-auto-update --patch-mode AutomaticByPlatform --nics $Context.ServiceNicName  | Out-Null
```

Comment:
I don't see the --enable-auto-update flag on the create for the Tunnel VM, does it need to be there? Does it need to be here if it is absent there?

Reply:
The "--enable-auto-update --patch-mode AutomaticByPlatform" occurs three times in the code: once in New-TunnelVM and twice in New-ServiceVM. It is there on all calls to "az vm create".